src/operators.jl

```diff
@@ -183,8 +185,55 @@ end
 function *{T<:Real}(g::DiGraph, v::Vector{T})
```

Comment:
I wonder if this might be cleaner as a context-manager/do-block.
```
if_installed(:GraphMatrices) do
   ...
end
```

I think maybe this could be done with:
```
function if_installed(f::Function, m::Symbol)
    try
        eval(Expr(:toplevel, Expr(:using, symbol(m))))
    catch
        return nothing
    end
    f()
end
```